app/templates/components/handle-link.hbs

```diff
@@ -1,5 +1,5 @@
-{{#if path}}
+{{#if useLinkTo}}
```

Comment:
Est-ce que le but de ce if c'est vraiment de savoir si on utilise un `linkTo` ?

Reply:
Oui tout à fait

Reply:
Quel est ton besoin métier ?

Reply:
Le besoin métier est que si c'est un lien interne on utilise un link-to ça évite de recharger l'application à chaque changement de page, sinon on créer une balise d'ancrage de façon classique. Par exemple dans le footer on a `Le code source est libre` qui renvoie vers le github